packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -0,0 +1,632 @@
+import { BigNumber } from 'bignumber.js'
+import * as React from 'react'
+import { StyleSheet, Text as RNText, View } from 'react-native'
+import CopyToClipboard from 'src/dev/CopyToClipboard'
+import ProgressCutBar from 'src/dev/ProgressCutBar'
+import { H1 } from 'src/fonts/Fonts'
+import { I18nProps, withNamespaces } from 'src/i18n'
+import Chevron, { Direction } from 'src/icons/chevron'
+import { HEADER_HEIGHT } from 'src/shared/Styles'
+import { colors, standardStyles, textStyles, typeFaces } from 'src/styles'
+import { cutAddress, formatNumber, weiToDecimal } from 'src/utils/utils'
+
+class Text extends RNText {
+  render() {
+    return <RNText style={[styles.defaultText, this.props.style]}>{this.props.children}</RNText>
+  }
+}
+
+interface HeaderCellProps {
+  style: any[]
+  name: string
+  order: boolean | null
+  onClick: () => void
+}
+
+const HeaderCell = React.memo(function HeaderCellFn({
+  style,
+  name,
+  order,
+  onClick,
+}: HeaderCellProps) {
+  return (
+    <View onClick={onClick} style={[styles.tableHeaderCell, ...((style || []) as any)]}>
+      <Text>{name}</Text>
+      <Text
+        style={[
+          styles.tableHeaderCellArrow,
+          ...(order !== null ? [styles.tableHeaderCellArrowVisible] : []),
+        ]}
+      >
+        <Chevron direction={order ? Direction.up : Direction.down} color={colors.white} size={10} />
+      </Text>
+    </View>
+  )
+})
+
+interface ValidatorsListProps {
+  data: any
+  isLoading: boolean
+}
+
+type orderByTypes = 'name' | 'total' | 'votes' | 'gold' | 'commision' | 'rewards' | 'uptime'
+
+export interface State {
+  expanded: number | undefined
+  orderBy: orderByTypes
+  orderAsc: boolean
+}
+
+class ValidatorsListApp extends React.PureComponent<ValidatorsListProps & I18nProps, State> {
+  state = {
+    expanded: undefined,
+    orderBy: 'name' as orderByTypes,
+    orderAsc: true,
+  }
+  private orderAccessors = {
+    name: (_) => _.name.toLowerCase(),
+    total: (_) => _.numMembers * 1000 + _.elected,
+    votes: (_) => +_.votesAbsolute,
```

Comment:
curious about the `+` here 

whats it do?

Reply:
The sorting required to sort by number of members and then by elected, so:

- 3 total & 1 elected = 3001
- 3 total & 2 elected = 3002
- 2 total & 2 elected = 2002

A easy way to sort it without 2 sorting functions or checking 2 conditions and the same time

Reply:
ah i meant the `+_.votesAbsolute`

Reply:
ive seen the `+` prefix a few places in this PR but not really before that so im curious